docs/guide/static-deploy.md

```diff
@@ -10,12 +10,12 @@ The following guides are based on some shared assumptions:
 {
   "scripts": {
     "build": "vite build",
-    "preview": "vite preview"
+    "serve": "vite preview"
   }
 }
 ```
 
-It is important to note that `vite preview` is intended for previewing the build locally and not meant as a production server.
+It is important to note that `vite serve` is intended for previewing the build locally and not meant as a production server.
```

Comment:
This should stay as vite preview

Reply:
Good catch, I  accidentally changed that one.